sdks/java/harness/build.gradle

```diff
@@ -24,7 +24,16 @@ applyJavaNature(shadowClosure: DEFAULT_SHADOW_CLOSURE <<
   // jars.
   {
     dependencies {
-      include('**/*.class')
+      //Directly include all depended projects
+      include(project(":beam-model-pipeline"))
```

Comment:
We want to use the shadow configuration so we'll need to specify these as `path: projectName, configuration: "shadow"`

We should also use a list containing all the projects and loop over it here and loop over it down below where we add the compile dependencies. This will allow for a person to update the list in one location and have it apply to both the shadow plugin and also the dependency set.